pkg/service/manipulator.go

```diff
@@ -82,6 +83,11 @@ func (m *manipulator) Process(spec ProcessSpec) ([]byte, error) {
 		data = m.processor.GrayScale(data)
 		trackDuration(grayScaleDurationKey, t, spec)
 	}
+	if CleanFloat(params[blur], 1000) > 0 {
+		t = time.Now()
+		data = m.processor.Blur(data, CleanFloat(params[blur], 1000))
```

Comment:
if you think previous review is good, this can be changed to:
`data = m.processor.Blur(data, radius)`